content/docs/ops/bosh-releases-in-15-minutes-with-ubuntu.md

```diff
@@ -0,0 +1,143 @@
+---
+menu:
+  docs:
+    parent: deployment
+title: Become a BOSH Developer in 15 Minutes using Ubuntu
+linktitle: Ubuntu
+---
+
+# How to Become a BOSH Developer in 15 Minutes
+The following describes how to start from a clean Ubuntu 14.04 install, and results in a Vagrant BOSH-lite instance running on top of VirtualBox.  This can be done both graphically, or through SSH exclusively.
+
```

Comment:
Maybe we could just point to the docs for installing bosh-lite. it already comes with a Vagrant file and is super easy to use.